modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/capability/xml/TestAnotherDocumentedConfig.java

```diff
@@ -14,7 +14,7 @@
 /**
  * This is some Another Config documentation.
  */
-@Configuration(name = "anotherConfig")
+@Configuration(name = "another")
```

Comment:
we should leave at least one case that tests for `another-config-config` not happening